sycl/source/handler.cpp

```diff
@@ -83,6 +82,218 @@ void *getValueFromDynamicParameter(
   return sycl::detail::getSyclObjImpl(DynamicParamBase)->getValue();
 }
 
+// Bindless image helpers
+
+// Fill image type and return depth or array_size
+unsigned int
```

Comment:
```suggestion
static unsigned int
```
Same for the rest of the helper functions in this file.

Reply:
Added to all functions except for `isDeviceGlobalUsedInKernel` as it is defined as an external function.